lib/cache/flags.ts

```diff
@@ -108,3 +108,13 @@ const checkMulti = async (keys: string[]): Promise<{ [k: string]: boolean }> =>
 
   return Object.fromEntries(mapped);
 };
+
+export type Flags = {
+  [k: string]: boolean;
+};
+
+// NOTE: Is there any security risk in accessing all our feature flags?
+// Alternatively this could become a static "safe list" of public feature flags that is updated manually.
+export async function getAllFlags(): Promise<Flags> {
```

Comment:
Thinking it might be better to pass in an array of flags we want to use ['addressComplete', 'repeatingSets', 'groups'] etc...

Reply:
That would work to. I just worry about maintaining that array of flags. Or maybe I'm over thinking it?

Reply:
Thinking... actually on second thought I like the idea of an array of flags passed in and that list maintained E.g. in the layout/page level.

Reply:
I'm trying to think.. Assuming there is no risk in exposing feature flag names, is there a benefit to choosing which feature flags? 
E.g. imagine a developer is adding a new feature and wonders why that feature flag isn't in the global list of feature flags. I'm sure s/he'd figure it out but it could be frustrating :)

Reply:
Think this depends a bit 

i.e if we wrap the provider around the front facing form (which we'll be doing) ... we don't need to expose the values of all our flags.

Reply:
i.e. does this open up issues 

Can I get access to feature via dev tools
https://github.com/user-attachments/assets/21c93672-ff71-4a6b-be7e-d3050e89b63c

Reply:
I wonder if all the feature flag names and status could become some kind of an attack vector? I can't think of anything but I guess it's safer to hide as much info as possible (the Java programmer side in me is in full agreement :)
Ok, a safe-list but then the client team will need to remember this. So for each use of the Provider (in each page/layout)

Reply:
Yup looks like the provider name and values are obfuscated

Dev
![1Screenshot 2024-09-23 at 9 24 49 AM](https://github.com/user-attachments/assets/c4ec17ed-7ebd-4862-8092-c624c896fd76)

vs. Staging
![2Screenshot 2024-09-23 at 9 25 03 AM](https://github.com/user-attachments/assets/08fea4b0-1011-45ba-b784-b516eec9f61d)